script.js

```diff
@@ -0,0 +1,12 @@
+function sum() {
+    let a = document.getElementById("firstNumber");
```

Comment:
Always a good idea to name things what they are, so in this case I'd go with something like `num1` or `firstNum` rather than `a` just to make it a little more readable.